tests/functional/functional_tests.py

```diff
@@ -3,30 +3,26 @@
 """Carry out functional tests on OneZoom pages using an automated browser via selenium
 
  Example: carry out all tests
-    nosetests -w ./ tests/functional
+    python -m pytest ./ tests/functional
  Example: carry out all tests for unsponsorable sites (museum displays)
-    nosetests -vs functional/sponsorship/test_unsponsorable_site.py
+    python -m pytest -s functional/sponsorship/test_unsponsorable_site.py
  Example: carry out test that unsponsorable sites give the correct page for invalid otts 
-    nosetests -vs functional/sponsorship/test_unsponsorable_site.py:TestUnsponsorableSite.test_invalid
-    
- If you have installed the 'rednose' package (pip3 install rednose), you can get nicer output by e.g.
- 
-    nosetests -vs ./tests/functional --rednose
+    python -m pytest -s functional/sponsorship/test_unsponsorable_site.py:TestUnsponsorableSite.test_invalid
     
  To carry out tests on a remote machine, you can specify a [url][server] and [url][port]
  in a config file, which will not give the FunctionalTest class an is_local attribute
  and hence will skip tests marked @attr('is_local'). E.g. for testing beta.onezoom.org, you can do
- 
```

Comment:
We'll want to choose a solution for running the functional tests on beta.onezoom.org, though it doesn't have to look like this

Reply:
Good point. We should record this in a README somewhere.